themes/digital.gov/layouts/shortcodes/checklist.html

```diff
@@ -0,0 +1,34 @@
+{{/* 
+  
+  Sample code for creating a checklist with a list of checkbox items
+ 
+  1. Use the checklist shortcode when you want to have a printable checklist
+  2. Provide a "header" and the checklist will set a gray bar beneath an H2
+  3. Nest {{< checkbox >}} shortcodes within the {{< checklist >}} to create a list of checkbox items
+  4. Use {{< checkbox-sublist-item sublist-item="sublist bullet point">}} will create a bullet point sublist item
+  5. Follow the same nesting structure as below, don't indent {{< checkbox-sublist-item >}} in the {{< checkbox >}} shortcode, else it will break
+
+  {{< checklist header="Preparing your presentation" >}}
+
+    {{< checkbox item="Item number 1 goes here" >}}
+    {{< checkbox-sublist-item sublist-item="detail one" >}}
+    {{< checkbox-sublist-item sublist-item="detail two" >}} 
+    {{< /checkbox >}}
+
+    {{< checkbox item="Item number 2 goes here, with supporting lines that wrap to the next if this is long enough" >}}  {{< /checkbox >}}
+    {{< checkbox item="Item number 3 goes here" >}}{{< /checkbox >}}
+  
+  {{< /checklist >}} 
```

Comment:
Indents seem off. The closing `{{</checklist >}}` is on the same level as the checkbox items.

Reply:
Yes, this is a hugo quirk. If they are indented then it will render as `pre` formatted code blocks. This is the standard way to get this to render, see https://stackoverflow.com/questions/72984744/custom-nested-hugo-shortcode-inner-markdownify-being-wrapper-in-pre